connection_windows.go

```diff
@@ -280,7 +280,19 @@ func (ai *AutomationItems) Remove(tag string) {
 	}
 	delete(ai.items, tag)
 }
-
+/*
+ * FIX: 
+ * some opc servers sometimes returns an int32 Quality, that produces panic
+ */
+func ensureInt16(q interface{}) int16 {
+	if v16, ok := q.(int16); ok {
+		return v16
+	}
+	if v32, ok := q.(int32); ok && v32 >= 32768 && v32 < 32768 {
```

Comment:
@resoftw thanks for the pr! I understand why you check whether the value is smaller than 32768 or not, but why do you also check for v32 >= 32768. This looks like it will never be true?

Reply:
oops... I'm sorry, typo, it should be **v32 >= -32768**

Reply:
Okay...